src/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+
```

Comment:
- [ ] `Required`
- [ ] Please enable the `disabled eslint` l because it defeats the purpose of installing the linters🥂